index.html

```diff
@@ -0,0 +1,458 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header class="header">
+      <h1 id="logo">Aziz Mejri</h1>
+
+      <nav class="nav">
+        <ul class="nav-menu desktop">
+          <li class="nav-item">
+            <a href="#home">Portfolio</a>
+          </li>
+          <li class="nav-item">
+            <a href="#about">About</a>
+          </li>
+          <li class="nav-item">
+            <a href="#contact">Contact</a>
+          </li>
+          <li class="nav-item desktop">
+            <a href="#mail">
+              <img src="./assets/images/icons/Icon-Mail.svg" alt="mail" />
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="hamburger mobile">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </button>
```

Comment:
Great job 💪  using a `button` tag as a wrapper for the hamburger menu to add tabbing feature which makes this icon accessible.
In conjunction with providing an accessible app to the clients, I suggest you include an `aria-controls` [attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-controls#:~:text=When%20an%20interactive,the%20impacted%20behavior.) to make it even clearer for users who depend on screen readers

example

```
<button type="button" class="hamburger mobile" aria-controls="menu">
          <span class="bar"></span>
          <span class="bar"></span>
          <span class="bar"></span>
 </button>
```